### PR TITLE
landing page font sizes

### PR DESCRIPTION
**Pull request by tinglinn**
increased font sizes, added bottom border to header

### CODE REVIEW

File: src/components/BreakingNews.tsx [hunk 1]
```diff
@@ -30,6 +30,7 @@ const BreakingNews = ({ news, onClick }: Props) => {
     try {
       setSplitNews(JSON.parse(news.choices[0].message.content));
     } catch (error: any) {
+      console.log("error", error);
```
Comment (acui51) [inline src/components/BreakingNews.tsx]: remove

**Review by acui51 — CHANGES_REQUESTED** [review]
you're working off of a previous branch you already created so it's pushing the entirety of landing.tsx -- can you get rid of that from this PR. landing.tsx became index.tsx which you correctly edited

**Comment by tinglinn** [comment]
completed changes

**Review by acui51 — APPROVED** [review]
lgtm

**Pull request merged**